lib/builderator/config/defaults.rb

```diff
@@ -95,33 +98,75 @@ module Config
         end
       end
 
-      generator.gemfile.vagrant do |vagrant|
-        vagrant.install true
-        vagrant.version 'v1.8.0'
-      end
+      generator.project :default do |base|
+        base.ruby.version '2.1.5'
```

Comment:
Heh, now that I'm looking at it, this value isn't even in the Gemfile template :grimacing: ...